configs/config/pretrain/byol/byol_1gpu.yaml

```diff
@@ -0,0 +1,121 @@
+# @package _global_
+config:
+  MAX_ITER: 312785 # int(1000. * Training set length / 4096) , here set according to ImageNet ILSVRC-2012 dataset
+  WARMUP_ITERS: 3128 # int(10. * Training set length / 4096) , here set according to ImageNet ILSVRC-2012 dataset
+  VERBOSE: False
+  LOG_FREQUENCY: 10
+  TEST_ONLY: False
+  TEST_MODEL: False
+  SEED_VALUE: 1337
+  MULTI_PROCESSING_METHOD: forkserver
+  HOOKS:
+    PERF_STATS:
+      MONITOR_PERF_STATS: True
+      ROLLING_BTIME_FREQ: 313
+    TENSORBOARD_SETUP:
+        USE_TENSORBOARD: True
+        EXPERIMENT_LOG_DIR: "byol_reference"
+        LOG_PARAMS: False
+        FLUSH_EVERY_N_MIN: 20
+  DATA:
+    NUM_DATALOADER_WORKERS: 5
+    TRAIN:
+      DATA_SOURCES: [disk_folder]
+      DATASET_NAMES: [imagenet1k_folder]
+      BATCHSIZE_PER_REPLICA: 64
+      LABEL_TYPE: sample_index    # just an implementation detail. Label isn't used
+      TRANSFORMS:
+        - name: ImgReplicatePil
+          num_times: 2
+        - name: RandomResizedCrop
+          size: 224
+        - name: RandomHorizontalFlip
+          p: 0.5
+        - name: ImgPilColorDistortion
+          strength: 0.5
+        - name: ImgPilMultiCropRandomApply
+          transforms:
+            - name: ImgPilGaussianBlur
```

Comment:
I'm pretty sure this translates into a [0.5, and 0.05] 
instead of [1, 0.1] (as indicated by the paper) probability of Gaussian blur, I'm pretty sure we need to change p: 1 here. 

My reasoning: 

We are using torchvision.RandomApply in `img_pil_multicrop_random_apply.py`, we are passing in 1 and 0.1 as the probabilities of applying the transformation:
```
- name: ImgPilGaussianBlur 
              p: 0.5
              radius_min: 0.1
              radius_max: 2.0
```

Hence the probabilities of applying ImgPilGaussianBlur are 1 * 0.5 and 1 * 0.1 respectively. Can you verify this?

Reply:
Indeed looking through the paper at Page:17 Table 6, Gaussian Blur probability ranges between 1 and 0.1